test/core/binary.wast

```diff
@@ -51,6 +51,114 @@
 (assert_malformed (module binary "\00asm" "\01\00\00\00" "\81\00\01\00") "malformed section id")
 (assert_malformed (module binary "\00asm" "\01\00\00\00" "\ff\00\01\00") "malformed section id")
 
+;; Unsigned LEB128 can have non-minimal length
```

Comment:
Some tests might be duplicated in this file (it is like this in the master commit https://github.com/WebAssembly/spec/commit/7fa2f20a6df4cf1c114582c8cb60f5bfcdbf1be1 - probably result of resolveing some merge conflicts), but I haven't untangled these duplications yet.